src/client/java/teammates/client/scripts/DataMigrationForComments.java

```diff
@@ -9,9 +9,9 @@
 import javax.jdo.Query;
 
 import teammates.client.remoteapi.RemoteApiClient;
-import teammates.common.datatransfer.CommentParticipantType;
-import teammates.common.datatransfer.CommentStatus;
-import teammates.common.datatransfer.InstructorAttributes;
+import teammates.common.datatransfer.attributes.InstructorAttributes;
+import teammates.common.datatransfer.misc.CommentParticipantType;
```

Comment:
Perhaps the category `misc` should not be used, and anything not falling under `questions` and `attributes` should be in the `datatransfer` package instead. Something like:
```
teammates.common.datatransfer: uncategorised/others
                       .attributes: *Attributes
                       .questions: *QuestionDetails, *ResponseDetails
```
If anything, this minimizes the changeset of this PR, and there is no significant loss (if any) in organization. Do you have any opinion on this?